insights_messaging/appbuilder.py

```diff
@@ -133,8 +135,17 @@ def build_app(self):
         if log_config:
             logging.config.dictConfig(log_config)
         else:
+            handler = logging.StreamHandler(stream=sys.stdout)
+            handler.setFormatter(LogstashFormatterV1()) # to keep the same format with cloud env
```

Comment:
Why do we need `LogstashFormatter`? Is it to be consistent with other apps?

Reply:
As the product, use the logstashformatter and write the log to aws log service.  So I update the log format for local testing (without aws)